src/models/Scholarships.ts

```diff
@@ -72,14 +73,21 @@ class Scholarships extends FirestoreCollection<ScholarshipData> {
       query = query.orderBy('deadline', 'asc');
     }
     if (opts.authorId) {
-      query = query.where('authorId', '==', opts.authorId);
+      query = query.where('author.id', '==', opts.authorId);
     }
 
     const postProcessFilter = (s: FirestoreModel<ScholarshipData>) =>
       s.data.amount.intersectsRange(opts.minAmount, opts.maxAmount);
 
     return FirestoreCollection.list(query, postProcessFilter);
```

Comment:
I think both the `.get()` and `.list()` methods followed by `.save()` may no longer work because they return `FirestoreModel<ScholarshipData>` instead of `ScholarshipModel`. And it's hard to fix that atm :/

Leave a TODO comment here and file a GitHub issue to fix this (probably the overall base classes need fixing) and I'll take a look at it.